src/index.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const state = {
+  temp: 75,
+  city: 'Seattle',
+  lat: '47.6038321',
+  long: '-122.330062',
+};
```

Comment:
I love this! Building your own version of state will allow for you to better understand the inner workings of state in React!